src/common/tree/tree_node_serializer.py

```diff
@@ -183,7 +173,7 @@ def tree_node_from_dict(
             if not isinstance(children, list):
                 raise ValueError(
                     f"The attribute '{child_attribute.name}' on blueprint '{node.type}' "
-                    + f"should be a list, but was '{str(type(children))}'"
+                    + f"should be a list, but was '{type(children)!s}'"
```

Comment:
Pretty sure the !s is unecessary here.
Tested this in playground, the !s makes no difference in output.

Reply:
removed